parsers/asm.c

```diff
@@ -208,9 +208,12 @@ static void makeAsmTag (
 													 kind_for_directive);
 				break;
 			case K_PSUEDO_MACRO_END:
-				macro_tag = getEntryInCorkQueue (*lastMacroCorkIndex);
-				macro_tag->extensionFields.endLine = getInputLineNumber ();
-				*lastMacroCorkIndex = CORK_NIL;
+				if (*lastMacroCorkIndex != CORK_NIL)
+				{
+					macro_tag = getEntryInCorkQueue (*lastMacroCorkIndex);
+					macro_tag->extensionFields.endLine = getInputLineNumber ();
+					*lastMacroCorkIndex = CORK_NIL;
+				}
```

Comment:
A more solid fix could be
```C
				macro_tag = getEntryInCorkQueue (*lastMacroCorkIndex);
				if (macro_tag)
					macro_tag->extensionFields.endLine = getInputLineNumber ();
				*lastMacroCorkIndex = CORK_NIL;
```
so whether the entry exits or not it won't crash, even if `*lastMacroCorkIndex` references a non-existing one.

But I guess it's not very important as this shouldn't ever happen.